tutorials/Homology-Modelling/homology-modelling-tutorial.ipynb

```diff
@@ -27,22 +29,36 @@
     "\n",
     "- Are there any other reasons why you might not want to select a particular structure? I.e. inaccurate assignemnt of EM density - missing loops or regions. \n",
     "\n",
-    "We will use this jupyter notebook to explore this.  The idea of this tutorial is that you should be able to follow it through at your own pace.  If you are of a timetabled course, then there will be some demonstrator help available, but this tutorial has been designed such that you should be able to complete it without help, even if you get stuck.\n",
+    "We will use this jupyter notebook to explore this.  If you are following the OxCompBio course, then an introduction to python and juypter notebooks is the first session.  If not then you can either work your way through that (found elsewhere in this repo) or see this quick (7 minute) https://www.youtube.com/watch?v=jZ952vChhuI introduction.\n",
+    "\n",
+    "\n",
+    "The idea of this tutorial is that you should be able to follow it through at your own pace.  If you are of a timetabled course, then there will be some demonstrator help available, but this tutorial has been designed such that you should be able to complete it without help, even if you get stuck.\n",
     "\n",
     "**BEFORE YOU START - Please ensure you installed all the necessary software (modeller, jalview, pymol - via conda install)**\n",
     "\n",
     "- Most of software necessary can be very simply installed under the conda framework. Conda is available on windows, linux and mac.\n",
     "\n",
+    "- It will be necessary to obtain the (free) license key for modeller (see https://salilab.org/modeller/ for details).\n",
+    "\n",
     "- This is the first step if you have not already installed conda (or miniconda) on your laptop - Rather than repeating the instructions here, simply head over to https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html and pick up the relevant instructions for your operating system.\n",
     "\n",
     "- Next, install the following packages with conda\n",
     "\n",
+    "  `% conda config -add channels defaults` \n",
```

Comment:
As mentioned previously we should not make them do this on their base conda installation. The best approach here is to move the install instructions out of the notebook, in the mean time, I'd suggest we instead make them do this:

```
conda create -y --name homology-tutorial python=3.7
conda activate homology-tutorial
conda config --env --add channels bioconda
conda config --env --add channels conda-forge
conda install -c salilab modeller
....
```

By doing this you avoid polluting the base environment with unwanted channels (and therefore breaking things for users).

Reply:
Yes -in fact I just updated that - in fact so that it coordinates with the rest of the course.  Also now all works with windows I think (only a couple of places where its a bit clunky)